cicd/42c-conf-examples/mapping-and-discovery/42c-conf.yaml

```diff
@@ -19,8 +21,10 @@
 # with its contents.
```

Comment:
"In this example, the sample 'petstore.json' file is mapped and is uploaded to the platform updating the API with the UUID 'e7cd62ce-1ee9-4320-af33-8bd9519c6f48' with its contents." --> "In this example, the sample 'petstore.json' file is mapped, and the plugin uploads it to the platform updating the contents of the API with the UUID 'e7cd62ce-1ee9-4320-af33-8bd9519c6f48' with the contents of the file from the source code repository."